campusromero_openedx_extensions/custom_registration_form/models.py

```diff
@@ -18,7 +18,7 @@ class CustomFormFields(models.Model):
     Holds extra info to be, used during
     user registration as a form extension.
     """
-    user = models.OneToOneField(USER_MODEL, null=True)
+    user = models.OneToOneField(USER_MODEL, null=True, on_delete=models.PROTECT)
```

Comment:
Why `PROTECT` I'm pretty sure the default used to be `CASCADE`

Reply:
Yes, but it does not seem logical as a business model that if the extra fields are eliminated, the user is eliminated ([doc](https://docs.djangoproject.com/en/3.2/ref/models/fields/#arguments)), however, I can ask Jorge